src/abstractdataframe/reshape.jl

```diff
@@ -367,6 +384,15 @@ struct RepeatedVector{T} <: AbstractVector{T}
     outer::Int
 end
 
+Base.parent(v::RepeatedVector) = v.parent
+Base.levels(v::RepeatedVector) = levels(parent(v))
+CategoricalArrays.isordered(v::RepeatedVector) = isordered(parent(v))
+
+function CategoricalArrays._isordered(v::RepeatedVector)
```

Comment:
I don't really like depending on internal functions like this. We should be able to fix this in CategoricalArrays by dispatching on `AbstractArray{<:CategoricalValue}` instead. Anyway that really concerns a corner case (calling `CategoricalArray(df.variable)`), so that's not urgent.

Reply:
OK - so I remove it here and assume that CategoricalArrays.jl will fix this.
(it also touches `categorical(df.variable)` which is pretty common though).